src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -484,7 +486,7 @@ Set the Following variables:
 |===
 | *Variable name* | *Description* | *Type*
 | `MODEL_URL`
-| The URL of the model that needs to be deployed.
+| A valid URL specified by the user leading to the model that needs to be deployed.
```

Comment:
I suggest to replace 'leading' by 'referencing' or 'pointing'

Reply:
Ok.